### ISSUE TITLE
Installation process failure using Raspberry PI imager

### ISSUE THREAD

**Issue by phiGitHub** [home-assistant/operating-system#3393]
### Describe the issue you are experiencing

Installation process failure using Raspberry PI imager.
File error ?
![HassPi5downloaderr@2x](https://github.com/home-assistant/operating-system/assets/42291563/384e67da-5d44-4b05-b61d-6d49c9fd8a01)
![HassPi5error@2x](https://github.com/home-assistant/operating-system/assets/42291563/48710ea5-a91d-4ad6-a0c8-234ff50ccd6a)

### What operating system image do you use?

rpi5-64  (Raspberry Pi 5 64-bit OS)

### What version of Home Assistant Operating System is installed?

None - Setup file download error

### Did you upgrade the Operating System.

No

### Steps to reproduce the issue

using Raspberry PI imager as per "Install Home Assistant Operating System" tutorial

### Anything in the Supervisor logs that might be useful for us?

```txt
no logs, see screen captures here above
```

### Anything in the Host logs that might be useful for us?

```txt
no logs, see screen captures here above
```

**Comment by sairon** [comment]
This is not an HAOS issue, and some other people ran into it too, see here: https://forums.raspberrypi.com/viewtopic.php?t=367859

It may be fixed in macOS 14.5 as reported in a different issue but likely with the same root cause: https://github.com/Homebrew/homebrew-core/issues/158759#issuecomment-2123135458

Other way around would be to use a different tool or system for flashing the image. Nothing we can about it, sorry.

**Comment by phiGitHub** [comment]
fixed with Github direct download
thanks for swift reply & best regards
